app/page.tsx

```diff
@@ -8,29 +8,36 @@ import jobDescription from "@/public/images/job-description.png";
 
 export default function Home() {
   return (
-    <Container className="flex items-center justify-center h-screen">
-      <div>
-        <div className="flex gap-4">
+    <Container className="flex items-center justify-center h-screen gap-10">
```

Comment:
gap here doesn't currently do anything: ```suggestion
    <Container className="flex items-center justify-center h-screen">
```